decorator.rb

```diff
@@ -0,0 +1,32 @@
+class Nemeable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [Optional] You may modify this error like this: `raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`

Reply:
Noted, thank you!!